README.md

```diff
@@ -18,6 +18,12 @@ ENV['routerScroll'] = {
   scrollElement: '#mainScrollElement'
 };
 
+If your viewport is not a scrollable area, you can still scroll to an element on the page.  This is particularly useful if instead of scrolling to the top of the window, you want to scroll to the top of the main content area.
+
+ENV['routerScroll'] = {
+  targetElement: '#mainTargetElement'
```

Comment:
What is the difference in `targetElement` and `scrollElement`?

Reply:
Updated the readme.  Just to recap, `scrollElement` is if the thing you want to scrollTop to is an area on your page that generates a vertical scroll bar.  Target element is something on your page you want to scroll to (somewhere that has a y coordinate > 0)